templates/profiles/my_profile.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
```

Comment:
This is template for view that will be created in different PR, so I will comment it there, once created...